Fix json format for .eslintrc on README

abinoam opened the pull request:
The old format is not compatible with JSON.parse.
(rule is a Symbol on it, not a String)

So it doesn't work at pronto-eslint.

Fix mmozuras/pronto-eslint#4

mmozuras commented:
👍

sandlerr commented:
Hey @abinoam this code block is not intended to be JSON but Ruby - it is the value of the default configuration `Hash`. We can add a default .eslintrc JSON in addition to the current code block.

abinoam commented:
Thanks @sandlerr.

Probably because of my lack of experience in JS and ESLint itself, with added difficult by not being a native english speaker, I have had a hard time trying to make pronto-eslintrb to work with custom configuration from the .eslintrc.

Things I have found non-intuitive:
- The .eslintrc format. As the ESLint documentation says it could be JSON, YAML and JS. Only looking at the source code that I could figure out that is JSON.
- Why .eslintrc doesn't have a file extension (per format) as ESLint documentation tells so.
- ESLint says .eslintrc without file extension is deprecated
- I haven't guessed that the configuration (at the README) was a plain Hash. One hint could be syntax highlighting. As you use Ruby syntax highlighting throughout the README file and not at that specific point of the README file. Maybe the README could be more explicit about it.

Another thing that made me loose some time was, I was expecting that my options just got merged into a default one. That's not what happens. It just completely takes place. So, when I added a blank .eslintc just for testing pronto-eslintrb suddenly stopped to work.

http://eslint.org/docs/user-guide/configuring#configuration-file-formats

I'll try to push (force) and updated README for appreciation.

abinoam committed:
Improve README. Clarify options and .eslinrc file.

arich commented on README.md:
````diff
@@ -45,32 +45,65 @@ Eslintrb.lint(source, 'no-undef' => true)
 
-If you pass `:eslintrc` as option, `.eslintrc` file is loaded as option.
+```ruby
+Eslintrb.lint(source, :eslinrc)
````
Typo, this should be `:eslintrc`.

abinoam replied:
Hey @arich, I haven't noticed your comment. Sorry. Fixing it now. And Thank you!

abinoam commented:
Dear @ocke and @sandlerr ,

I have fixed the typo spoted by @arich.
I have rebased on master and pushed -f.
Could you review the changes and merge if you think is ok?

sandlerr commented:
:+1: thanks!
